test/OpenTelemetry.Tests/Metrics/MetricAPITest.cs

```diff
@@ -231,21 +231,46 @@ void ProcessExport(Batch<Metric> batch)
         [Fact]
         public void MultithreadedLongCounterTest()
         {
-            var metricItems = new List<Metric>();
-            var metricExporter = new TestExporter<Metric>(ProcessExport);
+            this.MultithreadedCounterTest(deltaLongValueUpdatedByEachCall);
+        }
 
-            void ProcessExport(Batch<Metric> batch)
+        [Fact]
```

Comment:
Is there a better way to invoke these templated test functions? I wanted to keep them in their own functions so the output would be separate and also so one failing wouldn't cause both to be considered failing.